Sources/Views/MessagesCollectionView.swift

```diff
@@ -46,9 +46,14 @@ open class MessagesCollectionView: UICollectionView {
     internal var showMessageTimestampOnSwipeLeft: Bool = false
 
     private var indexPathForLastItem: IndexPath? {
-        let lastSection = numberOfSections - 1
-        guard lastSection >= 0, numberOfItems(inSection: lastSection) > 0 else { return nil }
-        return IndexPath(item: numberOfItems(inSection: lastSection) - 1, section: lastSection)
+        guard numberOfSections > 0 else { return nil }
+        
+        for offset in 1...numberOfSections {
+            let section = numberOfSections - offset
+            let lastItem = numberOfItems(inSection: section) - 1
+            if lastItem >= 0 { return IndexPath(item: lastItem, section: section) }
```

Comment:
Can we put the return on new line here please? :)

Reply:
@Kaspik no problem, that has been changed now.